server/db/models/board.js

```diff
@@ -20,16 +20,16 @@ var Lane = mongoose.model('Lane')
 
 var boardSchema = new mongoose.Schema({
     name: String, //The name of the repo
-    githubID: {
+    githubId: {
         type: Number,
-        unique: true
+        //unique: true
```

Comment:
@ssarda77 Good catch! I meant that as a temp fix when I had issues dropping the database, I'll change it back.